docs/operations/upgrade.md

```diff
@@ -16,6 +20,53 @@ Cortex has done lot of cleanup in the configuration files, and you are strongly
 
 Following fields were removed from YAML configuration completely: `claim_on_rollout` (always true), `normalise_tokens` (always true).
 
+#### Test Your Config
+
+To see if your config needs to change, one way to quickly test is to download a 1.5.0 (or newer) binary from the [release page](https://github.com/grafana/loki/releases/tag/v1.5.0)
+
+Then run the binary providing your config file `./loki-linux-amd64 -config.file=myconfig.yaml`
+
+If there are configs which are no longer valid you will see errors immediately:
+
+```shell
+./loki -config.file=loki-local-config.yaml
+failed parsing config: loki-local-config.yaml: yaml: unmarshal errors:
+  line 35: field dynamodbconfig not found in type aws.StorageConfig
+```
+
+Referencing the [list of diffs](https://cortexmetrics.io/docs/changelog/#config-file-breaking-changes) I can see this config changed:
+
+```diff
+-  dynamodbconfig:
++  dynamodb:
+```
+
+Also several other AWS related configs changed and would need to udpate those as well.
+
+
+### Loki Docker Image User Changes
+
+To improve security concerns, in 1.5.0 the Docker container no longer runs the loki process as `root` and instead the process runs as user `loki` with UID `10001` and GID `10001`
+
+This may affect people in a couple ways:
+
+#### Loki Port
+
+If you are running Loki with a config that opens a port number above 1000 (which is the default, 3100 for HTTP and 9095 for GRPC) everything should work fine in regards to ports.
```

Comment:
I think our default port was changed to 8080:
```
# HTTP server listen port
[http_listen_port: <int> | default = 80]
```

Reply:
was this default actually changed or is this just the weave default? (If it's the weave default we can't really change this in a clean way)

The config file that ships in the docker image sets it to 3100 so it's the "default" in docker